src/dodal/devices/zocalo/zocalo_results.py

```diff
@@ -53,6 +53,26 @@ class ZocaloSource(str, Enum):
 
 
 class XrcResult(TypedDict):
+    """
+    Information about a diffracting centre.
+
+    NOTE: the coordinate systems of centre_of_mass and max_voxel/bounding_box are not
+        the same; centre_of_mass coordinates are continuous whereas max_voxel and bounding_box
+        coordinates are discrete.
+    Attributes:
+         centre_of_mass: The position of the centre of mass of the crystal, adjusted so that
+         grid box centres lie on integer grid coordinates, such that a 1x1x1 crystal detected in
+         a single grid box at 0, 0, 0, has c.o.m. of 0, 0, 0, not 0.5, 0.5, 0.5
+         max_voxel: Position of the maximum voxel, in integer coordinates
+         max_count: max count achieved in a single voxel for the crystal
+         n_voxels: Number of voxels in the diffracting centre
+         total_count: Total of above-threshold spot counts in the labelled voxels
+         bounding_box: The rectangular prism that bounds the crystal, expressed
+            as the volume of whole boxes as a half-open range i.e such that
+            p1 = (x1, y1, z1) <= p < p2 = (x2, y2, z2) and
+            p2 - p1 gives the dimensions in whole voxels.
```

Comment:
Also that `p1` and `p2` refer to the top left and bottom right of the prism

Reply:
I'm going to wait until I have more feedback on https://github.com/DiamondLightSource/python-dlstbx/pull/263 before I assert that 0,0 is at the top-left - it may not actually be true depending on where the snake scan starts!